lib/compress/zstd_opt.c

```diff
@@ -454,6 +465,59 @@ repcodes_t ZSTD_updateRep(U32 const rep[3], U32 const offset, U32 const ll0)
         memcpy(opt[pos].rep, &rep_, sizeof(rep_));            \
```

Comment:
Is `SET_PRICE()` used anywhere anymore, or can it be deleted?

Reply:
It can be deleted.